### ISSUE TITLE
Color contrast violation on components that are not available for user interaction.

### ISSUE THREAD

**Issue by PriyankaUofI** [ainspector/firefox-ainspector-wcag#63]
Is this violation based on any other success criteria other than 1.4.3?

As per 1.4.3, Disabled buttons are excluded from the 1.4.3 Contrast (Minimum) Success Criterion. Note the following passage from https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum: User Interface Components that are not available for user interaction (e.g., a disabled control in HTML) are not required to meet contrast requirements. An inactive user interface component is visible but not currently operable. An example would be a submit button at the bottom of a form that is visible but cannot be activated until all the required fields in the form are completed.

![image](https://github.com/opena11y/ainspector-for-firefox/assets/84582289/099905a3-419c-4ea8-bdfa-5bb091c4472c)

![image](https://github.com/opena11y/ainspector-for-firefox/assets/84582289/a530e1df-ea2e-4f15-87e3-61ba510fd278)

**Comment by jongund** [comment]
@PriyankaUofI 
Sorry for the delayed response, thank you for your feedback I will look into this issue.
